tools/theme/src/Theme.ts

```diff
@@ -533,6 +534,10 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
                 this.shadowRoot.appendChild(style);
             });
         }
+        this.trackedChildren.forEach((el) => {
```

Comment:
Would `trackedChildren` practically be anything other than SpectrumElements here? Should we change the type of that? Or, should we add extra gating in the callback to make sure we're working with a SpectrumElement, if there are other elements that could be gathered here?

Reply:
Yes we should check for an instance of the SpectrumElement to add the system property
```js
this.trackedChildren.forEach((el) => {
    if (el instanceof SpectrumElement) {
        el.spectrumDelegates.system = this.system;
    }
});
```